dotnet-format/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/dotnet/core/sdk
+RUN dotnet tool install -g dotnet-format
+ENV PATH="/root/.dotnet/tools:${PATH}"
+RUN mkdir -p /code
+WORKDIR /code
+#COPY files /
+CMD ["dotnet", "format", "--help"]
```

Comment:
It's `dotnet format` here, but `dotnet-format` in the info `command` :thinking:

Reply:
I'm not sure which is better. `dotnet-format` is the executable, but if you call `dotnet format`, `dotnet` will call `dotnet-format` and that's how it's documented here: https://github.com/dotnet/format#how-to-use